src/applications/disability-benefits/686c-674/config/chapters/report-add-a-spouse/current-spouse-marriage-history-details/currentSpouseMarriageHistoryDetails.js

```diff
@@ -15,17 +22,13 @@ export const uiSchema = {
           'ui:required': formData => formData.spouseWasMarriedBefore,
         },
       },
-      startLocation: {
-        'ui:title': 'Place of marriage to former spouse',
-        state: {
-          'ui:required': formData => formData.spouseWasMarriedBefore,
-          'ui:title': stateTitle,
-        },
-        city: {
-          'ui:required': formData => formData.spouseWasMarriedBefore,
-          'ui:title': cityTitle,
-        },
-      },
+      startLocation: locationUISchema(
+        'spouseMarriageHistory',
+        'startLocation',
+        true,
+        'Where were they married?',
```

Comment:
Should this be `Place of marriage to former spouse`, or is this a copy update from Jim?